extension-manifest-v3/src/background/index.js

```diff
@@ -22,9 +22,13 @@ import './serp.js';
 import './helpers.js';
 import './external.js';
 
+import './filters/custom-filters.js';
```

Comment:
how about we call the folder `adblocker` instead?

Reply:
I think it will be gone, as it might be not needed.

Reply:
This is WIP